iris-mpc-gpu/src/server/actor.rs

```diff
@@ -590,6 +590,10 @@ impl ServerActor {
         return_channel: oneshot::Sender<ServerJobResult>,
     ) -> eyre::Result<()> {
         let now = Instant::now();
+
+        // TODO: since we now moved this to the actor again, is this a performance
+        // bottleneck and should we parallelize this, etc?
+        let batch = PreprocessedBatchQuery::from(batch);
```

Comment:
let's add a quick bench for this? this it's not parallelized anyways, local bench should be somewhat representative

Reply:
It's not free. Maybe just throwing a few `par_iters` at it is good enough?

Reply:
par_iters actually slow this down for smaller batch sizes < 16.